test/java/seedu/addressbook/storage/StorageFileTest.java

```diff
@@ -0,0 +1,98 @@
+package seedu.addressbook.storage;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collections;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.tag.UniqueTagList;
+import seedu.addressbook.storage.StorageFile.InvalidStorageFilePathException;
+import seedu.addressbook.storage.StorageFile.StorageOperationException;
+
+public class StorageFileTest {
+    private static final String TEST_DATA_FOLDER = "./test/java/seedu/addressbook/storage";
```

Comment:
Not really. Putting `./` in front is putting a redundant path component. This is akin to constructing the string using `"test/" + "java/" + "seedu/" + "addressbook/" + "storage"` -- we still get the same string but the code is not succinct.